src/main/webapp/query-control.js

```diff
@@ -25,12 +25,9 @@ function getColor() {
 
 function clearPreviousResult() {
   document.getElementById('data-table').innerHTML = '';
-  document.getElementById('colors').style.display = 'none';
-  document.getElementById('edit-title').style.display = 'none';
   document.getElementById('year-slider').style.display = 'none';
+  document.getElementById('top-buttons').style.display = 'none';
   document.getElementById('census-link').style.display = 'none';
-  document.getElementById('toggle-data-btn').style.display = 'none';
```

Comment:
This button doesn't seem to hide on new queries

Reply:
Maybe I'm looking at an updated version but for me all of the buttons are hidden when the page loads

Reply:
I also can't seem to replicate what you found, Kyra. Could you try a hard refresh, and if it still persists, let me know exactly what you did to see that? Thanks!

Reply:
If I query something there's no census data on the the toggle data button shows: http://localhost:8080/#person-type=under-18&action=moved&location=04&year=2018